bertopic/_bertopic.py

```diff
@@ -2339,6 +2339,95 @@ def visualize_documents(self,
                                             width=width,
                                             height=height)
 
+    def visualize_document_datamap(self,
```

Comment:
Thanks for adhering to my (somewhat) unconventional method of accessing the plotting capabilities!